### DISCUSSION TITLE
?

### DISCUSSION THREAD

**Discussion by PlayGame666** [skyline-emu/skyline#1739] [Q&A]
I have a question. When the skyline is complete, it will be able to run many games in the same way, and I have a question, will my phone be able to run the New super Mario Bros Deluxe game? my phone on easy launches the previous parts, for example, new super mario bros wii on another emulator is played at 40-60 fps on a 3DS emulator similarly, is there any chance that New super Mario Bros Deluxe will run for me so that it even works at 20 fps. here are my phone specs: Snapdragon 732G, RAM 8GB, Adreno(TM) 618. Device name Redmi note 10 pro

**Comment by dima-xd** [comment]
When skyline will be completed, sd 732g will not exist

**Reply by PlayGame666** [reply]
what is sd 618

**Reply by ghost** [reply]
SD 732 has adreno driver 618

**Comment by Swizzlefowl** [comment]
Why are you commenting on every issue and discussion here? It really spams the GitHub can you not do that.
